src/components/WhatYouWillNeedDialog.svelte

```diff
@@ -24,12 +25,12 @@
 </script>
 
 <main>
-  <div class="w-200">
-    <h1 class="text-2xl font-bold pb-5">{$t(headingId)}</h1>
+  <div class="w-175">
+    <DialogHeading>{$t(headingId)}</DialogHeading>
     {#if subtitleId}
       <p>{$t(subtitleId)}</p>
     {/if}
-    <div class="inline-grid grid-cols-{items.length} gap-20 py-20 px-10">
+    <div class="inline-grid grid-cols-{items.length} gap-20 py-13 px-10">
```

Comment:
The text under the items looks a little squished for this dialog. 🤔  Maybe reduce gap between the items? And/or widen the width of the dialogs? Not sure what looks good, might need some playing around
<img width="500" alt="Screenshot 2024-01-05 at 17 16 27" src="https://github.com/microbit-foundation/ml-trainer/assets/145345672/c3d662e3-6645-46ae-83ae-9cabb99033fd">

Reply:
I've reduced the gap for now. I'm sure there will be further tweaks in the future. [a8cbb21](https://github.com/microbit-foundation/ml-trainer/pull/114/commits/a8cbb21029632e1c26b05ebdaa9b15aa76e8b686)